doc/design/architecture/README.md

```diff
@@ -115,6 +116,17 @@ A load step context is an unit of test step control. Functionality:
 Metrics aggregation and representation. The component is a singleton which was differentiated from the Load Step Context
 component. Many load step contexts may be associated with the single metrics manager.
 
+## 2.5. After-test data aggregation
+
+This chapter describes how data is aggregated after Mongoose has finished aload step. 
+In standalone mode there is nothing to be done as we synchronously write in the file specified by `--output-fiie` as we go.
+But with distributed mode we aggregates data from local temporary files from worker nodes as soon as worload is done. 
+It happens when `--item-output-file` or `--output-metrics-trace-persist` are specified (or both). 
+We read it by ~16Mb chunks (if there is enough data) and we synchronously put it in the aggregated file on the controller 
```

Comment:
I think it would be better not to use the old name "controller" (term from 3v), instead use "entry node" and maybe add a link to basic terms or a distributed mode document.